packages/compiler/src/config/config-interpolation.ts

```diff
@@ -97,10 +116,20 @@ export function resolveValues<T extends Record<string, unknown>>(
   const resolvedValues: Record<string, unknown> = {};
   const resolvingValues = new Set<string>();
 
-  function resolveValue(key: string) {
+  function resolveValue(key: string, valuePassed?: any) {
```

Comment:
would it here be better to take a key path(list of keys) to the value? not sure I fully get how `valuePassed` works

Reply:
thanks for the advice! I did that on my latest iteration